src/App.js

```diff
@@ -6,25 +6,30 @@ import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
-
-    <p>
-      <span>comments: </span>
-      {comments.length}
-    </p>
-
-    <p>
-      <span>Users: </span>
-      {users.length}
-    </p>
-  </div>
-);
+import PostList from './components/PostList/PostList';
+
+const App = () => {
+  const prepearedPosts = posts.map((post) => {
+    const author = users.find(user => user.id === post.userId);
+    const commentsArr = comments.filter(
+      comment => post.userId === comment.postId,
+    );
+
+    return {
+      ...post,
+      author,
+      commentsArr,
```

Comment:
```suggestion
      comments,
```
do you need `Arr` part?